lib/postgres.go

```diff
@@ -391,6 +392,19 @@ type PGMetadataDerivedKey struct {
 	AccessSignature  []byte                           `pg:",type:bytea"`
 }
 
+// PGMetadataDAOCoinLimitOrder represents DAOCoinLimitOrderMetadata
+type PGMetadataDAOCoinLimitOrder struct {
+	tableName struct{} `pg:"pg_metadata_dao_coin_limit_order"`
+
+	TransactionHash            *BlockHash `pg:",pk,type:bytea"`
+	DenominatedCoinType        uint32     `pg:",use_zero"`
+	DenominatedCoinCreatorPKID *PKID      `pg:",type:bytea"`
+	DAOCoinCreatorPKID         *PKID      `pg:",type:bytea"`
```

Comment:
```suggestion
	DAOCoinCreatorPKID         *PKID      `pg:"dao_coin_creator_pkid,type:bytea"`
```
Postgres infers the column names based on the capitalization of the field name. You can override the default behavior by specifying the name in the struct tag. Having PKID at the end is fine for whatever reason and is parsed as `_pkid` so no need to change those fields.